### PR TITLE
feat: [#20] Add support for --version

### LINKED ISSUE #20: Support --version
obb should have a command-line switch for outputting its version like Babashka.

### PR DESCRIPTION
**Pull request by zane**
Resolves #20.

### CODE REVIEW

**Comment by borkdude** [comment]
Shall we add a little test for this?

**Comment by zane** [comment]
Sure!

**Comment by zane** [comment]
How's that look?

File: bb/integration_tests.clj [hunk 1]
```diff
@@ -49,6 +49,12 @@
   (is (str/starts-with? (obb* "-e" (pr-str '(js/Application "Safari")))
                         "#org.babashka.obb/object-specifier")))
 
+(deftest version-test
+  (is (re-matches #"obb v[\d]+\.[\d]+\.[\d]+(\-SNAPSHOT)?\n"
+                  (-> (obb** "--version")
+                      check
+                      :err))))
```
Comment (borkdude) [inline bb/integration_tests.clj:56]: I would say --version should print to `stdout` as that is info that tools might want to parse.

Comment (zane) [reply bb/integration_tests.clj]: Changed!

**Pull request merged**